### ISSUE TITLE
why is the modernizr in dist /bower_components instead of  dist/scripts? 

### ISSUE THREAD

**Issue by boltex** [yeoman/generator-webapp_DEPRECATED#249]
I'm a bit new with yeoman,  Finished a website and been using 'grunt' to make a 'final' version.

 I've been looking at the dist folder : I thought that when using the 'grunt' command alone a distribution version of the site was compiled, and that components were minified and placed in dist/scripts or dist/scripts/vendor and that there would be no trace of the bower_components folder.. but it is there with only modernizr in it...  Is that intended or is it a bug? 

In other words: why is there a bower_components folder in dist for modernizr? shouldn't all scripts be in /scripts and all css in /styles ,,.. ?
